src/Symfony/Component/DependencyInjection/Compiler/MergeExtensionConfigurationPass.php

```diff
@@ -52,18 +52,26 @@ public function process(ContainerBuilder $container)
             }
             $config = $resolvingBag->resolveValue($config);
 
-            $tmpContainer = new ContainerBuilder($resolvingBag);
-            $tmpContainer->setResourceTracking($container->isTrackingResources());
-            $tmpContainer->addObjectResource($extension);
-            if ($extension instanceof ConfigurationExtensionInterface && null !== $configuration = $extension->getConfiguration($config, $tmpContainer)) {
-                $tmpContainer->addObjectResource($configuration);
-            }
+            try {
+                $tmpContainer = new ContainerBuilder($resolvingBag);
+                $tmpContainer->setResourceTracking($container->isTrackingResources());
+                $tmpContainer->addObjectResource($extension);
+                if ($extension instanceof ConfigurationExtensionInterface && null !== $configuration = $extension->getConfiguration($config, $tmpContainer)) {
+                    $tmpContainer->addObjectResource($configuration);
+                }
 
-            foreach ($exprLangProviders as $provider) {
-                $tmpContainer->addExpressionLanguageProvider($provider);
-            }
+                foreach ($exprLangProviders as $provider) {
+                    $tmpContainer->addExpressionLanguageProvider($provider);
+                }
 
-            $extension->load($config, $tmpContainer);
+                $extension->load($config, $tmpContainer);
+            } catch (\Exception $e) {
```

Comment:
Any chance to specify less general exception?

Reply:
Would be a bug. Could be "finally" in fact.